<launch>
    <arg name="regions" default="false"/> <!-- workspace (false) or regions (true)-->

        <node name="calculate_clco_regions" pkg="simple-reachability" type="calculate_constant_level_constant_orientation_regions" output="screen" if="$(arg regions)">
            <rosparam command="load" file="$(find simple-reachability)/config/constant_level_constant_orientation.yaml"/>
        </node>

        <node name="calculate_clco" pkg="simple-reachability" type="calculate_constant_level_constant_orientation_workspace" output="screen" unless="$(eval arg('regions') == 'false')">
            <rosparam command="load" file="$(find simple-reachability)/config/smooth.yaml"/>
        </node>

</launch>
